src/index.js

```diff
@@ -1,17 +1,13 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
 import './index.css';
-import App from './App';
-import reportWebVitals from './reportWebVitals';
+import App from './components/App';
+import {BrowserRouter} from 'react-router-dom';
```

Comment:
Smart to add in Router from the beginning vs having to do it retroactively like we did in Rancid Tomatillos